### ISSUE TITLE
control policy can be empty, is "sequences" a required attribute?

### ISSUE THREAD

**Issue by cmohorea** [CiscoDevNet/terraform-provider-sdwan#312]
I have an empty policy with just a default deny action:
![image](https://github.com/user-attachments/assets/22f147d3-11a5-4ba6-9094-378c95d25b3e)

`terraform import` takes it in just fine:
```
    {
      "mode": "managed",
      "type": "sdwan_custom_control_topology_policy_definition",
      "name": "CLOUD_GATEWAY_POLICY",
      "provider": "provider[\"registry.terraform.io/ciscodevnet/sdwan\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "default_action": "reject",
            "description": "Topology policy for Cloud Gateway routers",
            "id": "036a515b-bbad-4512-825d-9f4e5ec2bb88",
            "name": "CLOUD_GATEWAY_POLICY",
            "sequences": null,
            "type": null,
            "version": null
          },
          "sensitive_attributes": []
        }
      ]
    },
```
Corresponding resource:
```
resource "sdwan_custom_control_topology_policy_definition" "test_CLOUD_GATEWAY_POLICY" {
  name = "test_CLOUD_GATEWAY_POLICY"
  description = "Topology policy for Cloud Gateway routers"
  default_action = "reject"
}
```
However, it generates an error:
```
│ Error: Missing required argument
│ 
│   on sdwan-tf-import-main.tf line 11576, in resource "sdwan_custom_control_topology_policy_definition" "CLOUD_GATEWAY_POLICY":
│ 11576: resource "sdwan_custom_control_topology_policy_definition" "CLOUD_GATEWAY_POLICY" {
│ 
│ The argument "sequences" is required, but no definition was found.
```

If I add `  sequences = null` to config, getting this
```
│ Error: Missing Configuration for Required Attribute
│ 
│   with sdwan_custom_control_topology_policy_definition.test_CLOUD_GATEWAY_POLICY,
│   on sdwan-tf-import-main.tf line 25, in resource "sdwan_custom_control_topology_policy_definition" "test_CLOUD_GATEWAY_POLICY":
│   25:   sequences = null
│ 
│ Must set a configuration value for the sequences attribute as the provider has marked it as required.
│ 
│ Refer to the provider documentation or contact the provider developers for additional information about configurable attributes that are required.
```

**Comment by cmohorea** [comment]
it seems to be working with `sequences = []`, however, then `terraform import` should then should do it as `[]` not as `Null`
can it be fixed either way (making it not required seems more logical to me)

**Comment by seconroy** [comment]
Released in v0.5.0.
